bundle/artifacts/artifacts_test.go

```diff
@@ -50,6 +50,9 @@ func TestArtifactUploadForWorkspace(t *testing.T) {
 										{
 											Whl: "/Workspace/Users/foo@bar.com/mywheel.whl",
 										},
+										{
+											Requirements: "/Workspace/Users/foo@bar.com/requirements.txt",
```

Comment:
This does not really test anything useful. It should instead point to local requirements.txt and then later assert that this path is translated correctly to remote path

Reply:
Having such test would catch an error below where local to remote translation is not happening